packages/utils/src/run-prettier.ts

```diff
@@ -13,6 +12,16 @@ import logger from 'webpack-cli/lib/utils/logger';
 
 export function runPrettier(outputPath: string, source: string): void {
     let prettySource: string = source;
+
+    let prettier;
+    try {
+        // eslint-disable-next-line node/no-extraneous-require
+        prettier = require('prettier');
+    } catch (err) {
+        logger.warn('File may not be properly formatted, use prettier to format it correctly.');
```

Comment:
`use` could me misleading, maybe this?

```suggestion
        logger.warn('File is not properly formatted because you don't have prettier installed, you can either install it or format it manually');
```

maybe this?

Reply:
done